### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by johnpaulusfrancia**
this pr contains some initial back end code. we have a simple function that can handle a GET request. We can run it on local with `serverless offline`.

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+service: myService
```
Comment (SierraChapman) [inline serverless.yml]:
```suggestion
service: hydronudge
```

File: serverless.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+provider:
+  name: aws
+  runtime: nodejs14.x
+
+functions:
+  plants:
+    handler: src/index.plants
```
Comment (SierraChapman) [inline serverless.yml]:
```suggestion
    handler: src/index.handler
```

File: tsconfig.json
Comment (SierraChapman) [inline tsconfig.json:1]: Get rid of comments?

**Review by SierraChapman — CHANGES_REQUESTED** [review]

File: serverless.yml (hunk 1 above)
Comment (johnpaulusfrancia) [reply serverless.yml]: done

File: serverless.yml (hunk 2 above)
Comment (johnpaulusfrancia) [reply serverless.yml]: done

**Review by SierraChapman — APPROVED** [review]

**Pull request merged**